yaks-html/lib/yaks/format/html.rb

```diff
@@ -39,8 +39,8 @@ def render_resource(resource, templ = section('resource'))
           .replace('.forms') {|div| render_forms(resource.forms).call(div) }
           .replace('.subresource') {|sub_templ| render_subresources(resource, templ, sub_templ) }
       end
-      alias render_collection_resource render_resource
-      alias render_null_resource render_resource
+      alias_method :render_collection_resource, :render_resource
+      alias_method :render_null_resource, :render_resource
```

Comment:
I personally perfer `alias`, but I never dug into the _lexical scope of `alias` leads to unpredictability in these cases_ argument. I switched to `alias_method` in my projects to
* have a smaller `.rubocop.yml`,
* be able to redefine `alias_method` (which is a method call rather than a keyword), and
* not have to remember the order of arguments (it’s alias + method in `alias_method`). :)

I’m happy to revert this one if you prefer `alias`!